README.md

```diff
@@ -57,7 +57,8 @@ The basic command to run a baseline scan would look like:
 | `--authUsernameField` | None | Username input id to use in auth apps |
 | `--authPasswordField` | None | Password input id to use in auth apps |
 | `--authSubmitField` | None | Submit button id to use in auth apps |
```

Comment:
let's make authSubmitField the first one, and the second one authPasswordSubmitField

if authPasswordSubmitField is set, you can assume multi-step (document it) and not need the authFormType parameter